specs/online_order_spec.rb

```diff
@@ -55,17 +73,41 @@
       # TODO: Your test code here!
       # Useful checks might include:
       #   - OnlineOrder.all returns an array
+      it "Returns an array object" do
```

Comment:
these tests don't run because you accidentally nested them into the `it` block started on line 54. In minitest, you can nest `describe` blocks into each other, but not multiple `it` blocks